client/src/components/pages/Settings.tsx

```diff
@@ -61,6 +61,7 @@ const Settings: FC = () => {
 
   const {user, signOutAsync} = useAuthContext();
   const {theme} = useTheme();
+
```

Comment:
We can remove the line break for one line variables.

Reply:
Thanks for your advice! 😄 

@JeffGuKang